### PR TITLE
osbuild2: new stage authconfig 

### PR DESCRIPTION
**Pull request by msehnout**
This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

File: osbuild-composer.spec [hunk 1]
```diff
@@ -326,7 +326,7 @@ The core osbuild-composer binary. This is suitable both for spawning in containe
 Summary:    The worker for osbuild-composer
 Requires:   systemd
 Requires:   qemu-img
-Requires:   osbuild >= 37
```
Comment (ondrejbudai) [inline osbuild-composer.spec:329]: This is in conflict with #1985.

**Review by thozza — APPROVED** [review]
Looks good 👍

**Comment by thozza** [comment]
@msehnout this PR needs rebasing. This will become an issue with all of your PRs adding support for new stages. I would suggest to either add them all to a single PR or rebase them on top of each other in some order and then merge in that order to not force you to rebase after each merge.

**Comment by msehnout** [comment]
> @msehnout this PR needs rebasing. This will become an issue with all of your PRs adding support for new stages. I would suggest to either add them all to a single PR or rebase them on top of each other in some order and then merge in that order to not force you to rebase after each merge.

Yes, I know. My plan is to merge them into a single PR for simplicity.

**Comment by msehnout** [comment]
Thanks for the reviews, I've merged my PRs into a single one. Please have a look at #2006.

**Pull request closed without merging**